src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -348,7 +348,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 CollectTupleFieldMemberNames(name, i + 1, numElements, elementNames)
 
-                Dim boundArgument As BoundExpression = BindRValue(argumentSyntax.Expression, diagnostics)
+                Dim boundArgument As BoundExpression = BindValue(argumentSyntax.Expression, diagnostics)
```

Comment:
Terminology is a bit confusing here. BindValue will enforce that the expression is an RValue (as you think about it), but will not enforce that it has a type.

VB will reclassify Values into RValues when they are being read and that will force the expressions into their default type, unless it is already transformed into an RValue by that time because of target typing conversions.